psardecrypt.cpp

```diff
@@ -12,11 +12,12 @@ int
 main(int argc, char *argv[]) {
 	int outSize;
 
-	if (argc != 2) {
-		printf("Usage: %s <infile>\n", basename(argv[0]));
+	if (argc != 3) {
+		printf("Usage: %s <infile> <outdir> \n", basename(argv[0]));
```

Comment:
We should make this optional instead of mandatory

Reply:
No idea on how to make it optional tho. i assume it'd require a lot of changes in order to make it work

Reply:
Ok, i have used a clever trick to make it optional, it'll work this way i think.